Makefile

```diff
@@ -52,38 +52,31 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 
 lint: ## check style with flake8
-	pipenv run flake8 koji_wrapper tests
+	pipenv run tox -eflake8
 
 test: ## run tests quickly with the default Python
-	pipenv run py.test tests
+	pipenv run  tox -epy
 
 test-all: ## run tests on every Python version with tox
 	pipenv run tox
 
 coverage: ## check code coverage quickly with the default Python
-	pipenv run coverage run --source koji_wrapper -m pytest
-	pipenv run coverage report -m
-	pipenv run coverage html
-	$(BROWSER) htmlcov/index.html
+	pipenv run  tox -epy
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/koji_wrapper.rst
-	rm -f docs/modules.rst
-	pipenv run sphinx-apidoc -o docs/ koji_wrapper
+	pipenv run sphinx-apidoc -o docs/ koji_wrapper ##TODO: remove
```

Comment:
Any reason for the TODO?

Reply:
I guess I could have left that out of the commit, the change is happening in the upcoming commit where I update the docs to match other projects.